From 03cf4b2ba8810323e633e4e6708419c7be3a5030 Mon Sep 17 00:00:00 2001
From: Martin Klapetek <mklapetek@kde.org>
Date: Thu, 7 Jan 2016 00:46:50 -0500
Subject: [PATCH 2/6] [notifications] Replace the mainItem's
 Layout.max/minWidth with fixed width

The item is not a layout and is not in a layout, so setting
Layout.min/maxWidth on it has no effect.
---
 applets/notifications/package/contents/ui/NotificationPopup.qml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/applets/notifications/package/contents/ui/NotificationPopup.qml b/applets/notifications/package/contents/ui/NotificationPopup.qml
index d398027..06a62be 100644
--- a/applets/notifications/package/contents/ui/NotificationPopup.qml
+++ b/applets/notifications/package/contents/ui/NotificationPopup.qml
@@ -67,12 +67,11 @@ PlasmaCore.Dialog {
 
     mainItem: MouseArea {
         id: root
-        Layout.minimumWidth: notificationItem.width + (units.smallSpacing * 2)
-        Layout.maximumWidth: Layout.minimumWidth
         LayoutMirroring.enabled: Qt.application.layoutDirection === Qt.RightToLeft
         LayoutMirroring.childrenInherit: true
 
         height: notificationItem.implicitHeight + (units.smallSpacing * 2)
+        width: notificationItem.width + (units.smallSpacing * 2)
 
         hoverEnabled: true
 
-- 
2.5.0

